src/tools/bin/plotRadiation

```diff
@@ -420,3 +477,5 @@ for myfile in args.path2Data:
     myfile.close()
 
     print(filename + "\t Done")
+
+#  LocalWords:  PIConGPU
```

Comment:
interesting but a bit random to put it in one file
one could think about a per-project or per-repository file but even there it's a bit... out of scope.

is it ok for you not to add it?